mapillm/exact_matching.py

```diff
@@ -0,0 +1,131 @@
+import os
+from mp_api.client import MPRester
+import sys
+
+
+class SuppressOutput:
+    #this is just because I'm annoyed with the print statements and progress bars from mp-api
```

Comment:
that's actually very good. I added a task to the backlog to use it in every MAPI queyr